test/api/test_datatypes.py

```diff
@@ -56,6 +57,19 @@ def test_converters( self ):
 
         assert found_fasta_to_tabular
 
+    def test_converter_present_after_toolbox_reload( self ):
+        response = self._get( "tools", data={'tool_id': 'CONVERTER_wiggle_to_interval_0'} )
```

Comment:
Can you use ``CONVERTER_fasta_to_tabular`` since it is actually referenced in ``test/functional/tools/sample_datatypes_conf.xml``? I think it should be available under more precise testing setups.